pallets/identity/src/lib.rs

```diff
@@ -207,7 +210,12 @@ decl_module! {
                 ExistenceRequirement::KeepAlive,
             )?;
 
-            let _new_id = Self::_register_did(sender, signing_items)?;
+            let new_id = Self::_register_did(sender, signing_items)?;
+            // Added for easier testing. To be removed before production
+            let cdd_providers = T::CddServiceProviders::get_members();
```

Comment:
Since Identities are only be supposed to be created by CDD providers, this whole function of register_did is there for testing only. I'll add a TODO for the whole function.